clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -779,6 +781,15 @@ class SingleDeviceFunctionTracker {
     if (!CurrentDecl)
       return;
 
+    /// This will print a uniqued call graph of functions being complied for the
```

Comment:
```suggestion
    /// This will print a uniqued call graph of functions being compiled for the
```
Can you clarify the sentence about what you mean with `uniqued` here?

Reply:
it is way too noisy to be useful or performant when there isn't any uniquing. and uniquing doesn't prevent to use this for what I usually use it for, identifying why some code was generated for device.